package.json

```diff
@@ -24,10 +24,12 @@
   "license": "MIT",
   "dependencies": {
     "@openactive/data-models": "^2.0.219",
```

Comment:
This needs updating once the Data Models PR is merged

Reply:
Done